bin/disco_alarms.py

```diff
@@ -6,12 +6,14 @@
     disco_alarms.py [--debug] [--dry-run] [--env ENV] update_notifications [--delete]
     disco_alarms.py [--debug] [--env ENV] update_metrics [--delete] --hostclass HOSTCLASS
     disco_alarms.py [--debug] [--env ENV] list [--hostclass HOSTCLASS]
+    disco_alarms.py [--debug] [--env ENV] delete_environment_alarms
```

Comment:
we already provide the environment `ENV` and we know this is for alarms. Why not just call it `delete` with an option of `--hostclass`?